pom.xml

```diff
@@ -21,7 +21,14 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter</artifactId>
+<!--			<artifactId>spring-boot-starter</artifactId>-->
+			<artifactId>spring-boot-starter-data-mongodb</artifactId>
+
+		</dependency>
+		<dependency>
```

Comment:
Is this dependency needed, since you have `spring-boot-started-data-mongodb`?

Reply:
i deleted the dependency just now and the program wouldnt run anymore

Reply:
Fair enough 👍